frontend/src/scenes/session-recordings/player/inspector/components/Timing/NetworkRequestTiming.tsx

```diff
@@ -23,8 +23,7 @@ export const NetworkRequestTiming = ({
 }): JSX.Element | null => {
     const [timelineMode, setTimelineMode] = useState<boolean>(true)
 
-    // if timeline view renders null then we fall back to table view
-    const timelineView = timelineMode ? <TimeLineView performanceEvent={performanceEvent} /> : null
+    const { isValid: isValidForTimelineView } = validateForTimelineView(performanceEvent)
```

Comment:
Not sure where this `performanceEvent` comes from but perhaps this processing is something we could do at a logic level and share the result across the waterfall and inspector timing views